src/LinearSolvers/LinearSolvers.jl

```diff
@@ -225,4 +227,69 @@ list_solvers(::KA.CPU) = [DirectSolver, Dqgmres, Bicgstab]
 Default linear solver on the CPU.
 """
 default_linear_solver(A::SparseMatrixCSC, device::KA.CPU) = DirectSolver(A)
+
+"""
+    CuDSSSolver <: AbstractLinearSolver
+
+Thin wrapper around a low-level `CudssSolver`. Does:
+  - first `ldiv!`:   "analysis" → "factorization" → "solve"
```

Comment:
I would suggest doing the symbolic analysis directly in the constructor `CuDSSSolver`